packages/importer/src/columns.ts

```diff
@@ -149,23 +153,80 @@ export const coerceValue = (
         ? { value: parseMultiple(String(value)), isError: false }
         : { value: null, isError: true };
     }
+    case 'file': {
+      const res = await parseFile((value as string).trim(), proxyFunction);
+      if (!res) return { value: null, isError: true };
+      return {
+        value: { name: 'upload', mediaType: res.mediaType, base64Content: res.base64Content } as XataFile,
+        isError: false
+      };
+    }
+    case 'file[]': {
+      // Cannot import multiple raw Base64 files because we will split in the wrong place
+      const promises = (value as string).split(/[,;|]/).map((uri) => parseFile(uri, proxyFunction));
```

Comment:
This is a limitation with the current code right now. Base64 values will have these characters in them so the split will be wrong if the user intends multiple files